packages/core/contracts/zodiac/OptimisticGovernor.sol

```diff
@@ -148,6 +150,7 @@ contract OptimisticGovernor is Module, Lockable {
     function setLiveness(uint64 _liveness) public onlyOwner {
         // Set liveness for disputing proposed transactions.
         require(_liveness > 0, "liveness can't be 0");
+        require(_liveness < 3144960000, "liveness can't be longer than 5200 weeks");
```

Comment:
you can just write `52 weeks` pretty sure

Reply:
https://docs.soliditylang.org/en/v0.8.10/units-and-global-variables.html#time-units